examples/machine-controller.yaml

```diff
@@ -10,6 +10,42 @@ spec:
     kind: Machine
     plural: machines
 ---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: machinesets.cluster.k8s.io
+spec:
+  group: cluster.k8s.io
+  version: v1alpha1
+  scope: Namespaced
+  names:
+    kind: MachineSet
+    plural: machinesets
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: machinedeployments.cluster.k8s.io
+spec:
+  group: cluster.k8s.io
+  version: v1alpha1
+  scope: Namespaced
+  names:
+    kind: MachineDeployment
+    plural: machinedeployments
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusters.cluster.k8s.io
```

Comment:
The reason for creating this and the `machinedeployments.cluster.k8s.io` crd is that the `SharedInfomers` controller starts informers for this in `startInformers` and spams the logs with errors if they are not present: https://github.com/kubermatic/machine-controller/pull/317/files#diff-261786ef9d08de00190759a59afc4763